### PR TITLE
WEB: standardized footer GH48012 

### LINKED ISSUE #48012: WEB: Standardize website/docs footer
At the moment we've got different footer notes for the website and the docs, and we're discussing adding our hosting sponsor there. Probably worth agreeing what footer we want to use, and standardize it all over.

**Current website footer:**

pandas is a fiscally sponsored project of [NumFOCUS.](https://numfocus.org/)

**Current docs footer:**

© Copyright 2008-2022, the pandas development team.
Created using [Sphinx](http://sphinx-doc.org/) 4.5.0.

**Proposal:**

© 2022 [NumFOCUS, Inc](https://numfocus.org/). Hosted by [OVH Cloud](https://us.ovhcloud.com/)

Personally I think this is as concise as possible, with the relevant information. For what I read, since many years it's not really required to have the copyright note in websites. But some sources still recommend it. While many projects use something like `Copyright 2022, the pandas team`, seems that doesn't make much sense from a legal point of view, as the pandas team is not an organization, and the copyright claim in this case is from NumFOCUS. I think it's nice to have them in the footer anyway.

Any better idea?

### PR DESCRIPTION
**Pull request by FrnRmn**
- [ ] closes #48012 
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

### CODE REVIEW

File: web/pandas/_templates/layout.html [hunk 1]
```diff
@@ -91,7 +91,7 @@
                 </li>
             </ul>
             <p>
-                pandas is a fiscally sponsored project of <a href="https://numfocus.org">NumFOCUS.</a>
+                © 2022 pandas via <a href="https://numfocus.org">NumFOCUS.</a>
```
Comment (datapythonista) [inline web/pandas/_templates/layout.html:94]: Can you use `NumFOCUS, Inc.` instead please, that's the legal name.

For the year, can you make it be generated dynamically, so we don't need to update it every year? I'm not sure if jinja allows direct access to the `datetime` module, or what the best option could be. Feel free to add the year to the context if you think that's the best, or also use javascript

**Review by datapythonista — COMMENTED** [review]
Thanks @FrnRmn for working on this, and sorry for the delay with the review.

I'd also make the changes to the docs footer here. I've seen you say on the ticket that you could only partially make them, but probably good enough for now. Thanks!

File: web/pandas/_templates/layout.html (hunk 1 above)
Comment (FrnRmn) [reply web/pandas/_templates/layout.html:94]: Thanks for the feedback, I'll try soon!

**Comment by FrnRmn** [comment]
Ups, I ended up updating the PR before I intended.  I wanted to merge the commits into a single commit with a better name :/ Sorry for that

**Comment by datapythonista** [comment]
> Ups, I ended up updating the PR before I intended. I wanted to merge the commits into a single commit with a better name :/ Sorry for that

Don't worry about it, we squash all the commits into one when we merge PRs, so it doesn't add much value to squash them before. And in some cases it's useful to have separate commits, so we don't need to review everything again, and can check just what changed.

Checking your changes, I'm wondering if it'd make sense to have the footer content for the website in the configuration yaml. I think we the whole config to the context, so not much change would be needed, except for the year. For it, a simple content preprocessor that replaces a string like `{year}` by the current year could be used. I think this could be a bit better than the javascript. But happy to use the javascript if you have a strong preference for it.

For Sphinx, is Sphinx adding the word `Copyright` automatically? That's annoying, quite a poor design decision in my opinion. If you want to have a quick look and see if there is an easy way to overwrite that, it would be great. But I wouldn't spend too much time on it, Sphinx code is a huge mess.

**Comment by datapythonista** [comment]
I opened #48543 to update the footer for the docs, as we want to have it merged before the release on Monday. I'll let you continue here with the update for the website, which is independent from the release.

**Comment by FrnRmn** [comment]
@datapythonista hi, sorry just wanted to let you know that I started working today and have a lot to do now. If it is something with a deadline please go ahead. Otherwise, I might take a while (like a couple of weeks) :)

**Comment by datapythonista** [comment]
No problem, thanks for letting us know. If I have the time to look at this before you, I'll let you know and close this PR. Thanks for the work on this!

**Comment by datapythonista** [comment]
Superseded by #48557

Thanks @FrnRmn for the work on this.

**Pull request closed without merging**